### PR TITLE
fix(Navigation): emojis in names [YTFRONT-4104]

### PR DESCRIPTION
**Pull request by SimbiozizV**
https://nda.ya.ru/t/WYvyvoCE75J3qJ

### CODE REVIEW

File: packages/ui/src/ui/store/actions/navigation/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ import {isSupportedEffectiveExpiration} from '../../../store/selectors/thor/supp
 export function updateView(settings = {}) {
     return (dispatch, getState) => {
         const state = getState();
-        const currentPath = getPath(state);
+        const currentPath = decodeURIComponent(getPath(state));
```
Comment (ma-efremoff) [inline packages/ui/src/ui/store/actions/navigation/index.js:42]: It looks like an unnecessary change. Do we really need it?

File: packages/ui/src/ui/store/actions/navigation/modals/create-directory.tsx [hunk 2]
```diff
@@ -11,13 +11,10 @@ import {
...
 } from '../../../../constants/navigation/modals/create-directory';
-import {
-    hideError,
-    showErrorInModal,
-} from '../../../../store/actions/navigation/modals/path-editing-popup';
+import {hideError, showErrorInModal} from './path-editing-popup';
 import {HIDE_ERROR} from '../../../../constants/navigation/modals/path-editing-popup';
 import CancelHelper from '../../../../utils/cancel-helper';
-import {RootState} from '../../../../store/reducers';
+import {RootState} from '../../../reducers';
```
Comment (ma-efremoff) [inline packages/ui/src/ui/store/actions/navigation/modals/create-directory.tsx:17]: All the changes looks unnecessary in this file. Were the changes forgotten to remove?

File: packages/ui/src/ui/store/actions/navigation/index.js (hunk 1 above)
Comment (SimbiozizV) [reply packages/ui/src/ui/store/actions/navigation/index.js:42]: yes, otherwise the links will not work correctly

File: packages/ui/src/ui/store/actions/navigation/modals/create-directory.tsx (hunk 2 above)
Comment (SimbiozizV) [reply packages/ui/src/ui/store/actions/navigation/modals/create-directory.tsx:17]: ide suggested a shorter route. There's nothing wrong with fixing it

**Review by ma-efremoff — APPROVED** [review]
